src/frontend/src/heartbeat.rs

```diff
@@ -93,6 +93,12 @@ impl HeartbeatTask {
                 match resp_stream.message().await {
                     Ok(Some(resp)) => {
                         debug!("Receiving heartbeat response: {:?}", resp);
+                        if let Some(msg) = resp.mailbox_message.as_ref() {
```

Comment:
The log may be annoying.

Reply:
removed in [96b4e41](https://github.com/GreptimeTeam/greptimedb/pull/5792/commits/96b4e410d09b34f5e5b4a679018bf68ab59652c6)